lib/fathead/mdnjs/parse.py

```diff
@@ -248,30 +248,34 @@ def writerows(self):
                         match = re.search('(?:.*\.)([^\(]+)(?:\(\))?', mdn.title)
                         # remove class_word: getAnimationFrame
                         strip_title = match.group(1)
-                        # check if not an Article
-                        if all(x in [keyword, strip_title] for x in self._writer.articles_index):
-                          return
+                        # skips redirect if already an article
+                        if strip_title.lower() not in self._writer.articles_index:
+                            self._writer.writerow({
+                              'title': strip_title,
+                              'type': 'R',
+                              'redirect': mdn.title
+                            })
```

Comment:
I think you need to add `strip_title.lower()` to the `articles_index` after this to prevent duplicates?